lib/msf/core/post/file.rb

```diff
@@ -537,19 +537,29 @@ def rm_rf(*remote_dirs)
   alias :dir_rm :rm_rf
 
   #
-  # Rename a remote file.
+  # Renames a remote file and returns true on success and false
+  # on failure
   #
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
   def rename_file(old_file, new_file)
+  	force = true
+  	force = false if directory?(new_file)
+    verification_token = Rex::Text.rand_text_alphanumeric(8)
     if session.type == "meterpreter"
-      return (session.fs.file.mv(old_file, new_file).result == 0)
-    else
-      if session.platform == 'windows'
-        cmd_exec(%Q|move /y "#{old_file}" "#{new_file}"|) =~ /moved/
-      else
-        cmd_exec(%Q|mv -f "#{old_file}" "#{new_file}"|).empty?
+      begin
+      	new_file = new_file + session.fs.file.separator + session.fs.file.basename(old_file) if directory?(new_file)
```

Comment:
Looks like a hard tab was used for the indentation here. Can you switch it back to two spaces?